### PR TITLE
Improves readability

### PR DESCRIPTION
**Pull request by imechemi**
#### Summary
An indentation was not proper in code sample. It falsely shows as nested element.

#### Motivation
Improve readability.

This PR…
-->
- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

**Review by wbamberg — APPROVED** [review]
This is much better, @imechemi !

There are more improvements we could make here:
* semicolons at the ends of lines
* use `for (const div of divs)` rather than `for (let i = 0; i < divs.length; i++)`
* use lowercase for function name initials (e.g. `onDivClick` rather than `OnDivClick`)

Let me know if you would like to make these changes too in this PR. Otherwise I'm happy to merge it as it is, since this is already an improvement.

**Comment by imechemi** [comment]
> This is much better, @imechemi !
> 
> There are more improvements we could make here:
> 
> * semicolons at the ends of lines
> * use `for (const div of divs)` rather than `for (let i = 0; i < divs.length; i++)`
> * use lowercase for function name initials (e.g. `onDivClick` rather than `OnDivClick`)
> 
> Let me know if you would like to make these changes too in this PR. Otherwise I'm happy to merge it as it is, since this is already an improvement.

@wbamberg I have updated as suggested except that have removed semicolons from all to keep it consistent. Let me know if this does not comply with mdn guide. Thanks for reviewing.

**Comment by wbamberg** [comment]
> > This is much better, @imechemi !
> > There are more improvements we could make here:
> > 
> > * semicolons at the ends of lines
> > * use `for (const div of divs)` rather than `for (let i = 0; i < divs.length; i++)`
> > * use lowercase for function name initials (e.g. `onDivClick` rather than `OnDivClick`)
> > 
> > Let me know if you would like to make these changes too in this PR. Otherwise I'm happy to merge it as it is, since this is already an improvement.
> 
> @wbamberg I have updated as suggested except that have removed semicolons from all to keep it consistent. Let me know if this does not comply with mdn guide. Thanks for reviewing.

Thank you @imechemi ! Please do add semicolons. This is part of our style guide: https://developer.mozilla.org/en-US/docs/MDN/Guidelines/Code_guidelines/JavaScript#use_expanded_syntax . I wouldn't generally ask to fix this but if the PR is fixing other code style issues, as this one it, then it's worth doing at the same time.

File: files/en-us/web/api/event/eventphase/index.md [hunk 1]
```diff
@@ -92,63 +92,61 @@ let clear = false,
     useCapture = false;
```
Comment (AnilSeervi) [inline files/en-us/web/api/event/eventphase/index.md:92]:
```suggestion
    chCapture = null;
```
The example doesn't use the `useCapture` variable.
`chCapture` isn't declared earlier in code, making it initialize globally at line 97.

**Review by AnilSeervi — COMMENTED** [review]
The example is now much more readable.
slight changes and it'll be awesome ✨

File: files/en-us/web/api/event/eventphase/index.md (hunk 1 above)
Comment (imechemi) [reply files/en-us/web/api/event/eventphase/index.md]: I have updated now.

**Review by wbamberg — APPROVED** [review]
Thank you @imechemi and @AnilSeervi for the review! I fixed a couple of errors that crept in, think this is good now. Much better than the original example, thanks very much @imechemi !

**Pull request merged**